install

```diff
@@ -632,6 +632,9 @@ case "${dist}-${code}" in
     debian-8|debian-jessie)
         code="jessie"
         is_debian_dist=true
+        echo -e "ERROR: Debian Jessie is archived and does not receive any security or other updates since 2018-05-17." >&2
+        echo -e "The LibreTime installer dropped support for installing LibreTime on Jessie in 3.0.0-alpha.8." >&2
```

Comment:
```suggestion
        echo -e "The LibreTime installer dropped support for installing LibreTime on Jessie in 3.0.0-alpha.7." >&2
```
I'd assume that this would make it into alpha.7... Not that it's that important.

LGTM

Reply:
My thought was warn for one release and then deprecate. So release alpha.7 simply warning about the change and then drop full support in alpha.8. I do need to fix the grammar though